### ISSUE TITLE
Resource aws_lb_listener_certificate doesn't support import

### ISSUE THREAD

**Issue by harj-the-dev** [hashicorp/terraform-provider-aws#13610]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Currently we are unable to `import `resource `aws_lb_listener_certificate`.
Error received:
```
resource aws_lb_listener_certificate doesn't support import
````
I would like to request support be added for `import` to support this resource.

### New or Affected Resource(s)

Possibly
* aws_lb_listener
* aws_lb
* aws_alb_listener
* aws_alb
* aws_alb_listener_certificate

### Potential Terraform Configuration

n/a

```hcl
data "aws_acm_certificate" "example_com" {
    domain      = "example.com"
    statuses    = ["ISSUED"]
    most_recent = "true"
}
resource "aws_lb_listener_certificate" "example_com" {
    listener_arn    = "##SOME_ARN_STRING##"
    certificate_arn = "${data.aws_acm_certificate.example_com.arn}"
}
```

### References

* https://www.terraform.io/docs/providers/aws/r/lb_listener_certificate.html
* https://docs.aws.amazon.com/cli/latest/reference/elbv2/describe-listener-certificates.html

**Comment by joaoluiznaufel** [comment]
does terraform supports this import?

**Comment by bflad** [comment]
Support for this functionality has been merged and will release with version 3.28.0 of the Terraform AWS Provider, later today. Thank you to @DrFaust92 for the implementation. 👍
